### ISSUE TITLE
Tracker bug for keybindings/layout issues

### ISSUE THREAD

**Issue by GoogleCodeExporter** [chrsmith/hedgewars#496]
```
Hooray!




System information:
Qt version: 4.7.4
Operating system: Mac OS X 10.6.8 "Snow Leopard"
Total RAM: 4096 MB
Available RAM: 134215318 MB
Size of the screen(s): 1440x900
Number of screens: 1
Processor: Intel(R) Core(TM)2 Duo CPU     T9550  @ 2.66GHz
Number of cores: 2
Compiler version: GCC 4.2.1 (Based on Apple Inc. build 5658) (LLVM build 
2335.15.00)
Compiler architecture: x86_64
Kernel: Darwin new-host-7.home 10.8.0 Darwin Kernel Version 10.8.0: Tue Jun  7 
16:33:36 PDT 2011; root:xnu-1504.15.3~1/RELEASE_I386 i386
```

Original issue reported on code.google.com by `feedback...@gmail.com` on 8 Dec 2012 at 3:48
* Blocked on: #138, #148, #155, #188, #479
* Merged into: #192

**Comment by GoogleCodeExporter** [comment]
```
[deleted comment]
```

**Comment by GoogleCodeExporter** [comment]
```
[deleted comment]
```

**Comment by GoogleCodeExporter** [comment]
```
maybe a few of these will be resolved using sdl2
```

Original comment by `vittorio...@gmail.com` on 25 Mar 2013 at 2:07
* Changed title: **Tracker bug for keybindings/layout issues**
* Changed state: **New**
* Added labels: **Type-Tracker**
* Now blocked on: #138, #148, #155

**Comment by GoogleCodeExporter** [comment]
Original comment by `vittorio...@gmail.com` on 27 Mar 2013 at 10:25
* Now blocked on: #188, #479

**Comment by GoogleCodeExporter** [comment]
Original comment by `vittorio...@gmail.com` on 16 May 2013 at 8:24
* Changed state: **Duplicate**
